### ISSUE TITLE
Mismatch for Kurento packages in BigBlueButton 2.0-RC9

### ISSUE THREAD

**Issue by ffdixon** [bigbluebutton/bigbluebutton#6638]
We mistakenly included later versions of Kurento in 2.0-RC9.  Here's a comparison between RC8 and RC9.

RC8 (HTML5 webcams work)
~~~
kurento-media-server | 6.7.0+201806281201.25.cc67c2e | https://ubuntu.bigbluebutton.org/xenial-200-rc8 bigbluebutton-xenial/main amd64 Packages
kms-core | 6.8.1+201811222103.28.5d99648 | https://ubuntu.bigbluebutton.org/xenial-200-rc8 bigbluebutton-xenial/main amd64 Packages
kms-elements | 6.8.0+201811222126.25.af1a776 | https://ubuntu.bigbluebutton.org/xenial-200-rc8 bigbluebutton-xenial/main amd64 Packages
kms-filters | 6.7.0+201806211640.18.d3dac3a | https://ubuntu.bigbluebutton.org/xenial-200-rc8 bigbluebutton-xenial/main amd64 Packages
~~~

RC9 (HTML5 webcams don't work)
~~~
kurento-media-server | 6.7.0+201812200318.28.cc67c2e | https://ubuntu.bigbluebutton.org/xenial-200 bigbluebutton-xenial/main amd64 Packages
kms-core | 6.9.1+201812211912.32.9049f74 | https://ubuntu.bigbluebutton.org/xenial-200 bigbluebutton-xenial/main amd64 Packages
kms-elements | 6.9.1+201812200310.28.a024284 | https://ubuntu.bigbluebutton.org/xenial-200 bigbluebutton-xenial/main amd64 Packages
kms-filters | 6.7.0+201812200315.21.d3dac3a | https://ubuntu.bigbluebutton.org/xenial-200 bigbluebutton-xenial/main amd64 Packages
~~~

**Comment by ffdixon** [comment]
Fixed in upcoming BigBlueButton 2.0-RC10

**Comment by ffdixon** [comment]
This is what the versions now look like in 2.0-RC10

~~~
www-data@droplet-109:/var/log/nginx$ dpkg -l | grep kms
ii  kms-core                              6.9.1.xenial~20190128192718.38.113574f        amd64        Kurento Core module
ii  kms-elements                          6.9.1.xenial~20190128193957.33.a024284        amd64        Kurento Elements module
ii  kms-filters                           6.9.1.xenial.20190128194440.28.92d67c6        amd64        Kurento Filters module
ii  kms-jsonrpc                           6.9.1.xenial~20181217205215.1.58eaf08         amd64        Kurento JSON-RPC library
ii  kmsjsoncpp                            1.6.3.xenial.20181017180019.d78deb7           amd64        Kurento jsoncpp library
~~~

All the KMS (Kurento Media Server) packages now match.
